cumulusci/cli/cci.py

```diff
@@ -136,34 +137,47 @@ def show_debug_info():
     pdb.post_mortem()
 
 
+def show_version_info():
+    console = rich.get_console()
+    console.print(f"CumulusCI version: {cumulusci.__version__} ({sys.argv[0]})")
+    console.print(f"Python version: {sys.version.split()[0]} ({sys.executable})")
+    console.print()
+
+    current_version = get_installed_version()
+    latest_version = get_latest_final_version()
+
+    if not latest_version > current_version:
+        console.print("You have the latest version of CumulusCI :sun_behind_cloud:\n")
+        display_release_notes_link(str(latest_version))
+        return
+
+    console.print(
+        f"[yellow]There is a newer version of CumulusCI available: {str(latest_version)}"
+    )
+    console.print(f"To upgrade, run `{get_cci_upgrade_command()}`")
+    display_release_notes_link(str(latest_version))
+
+
+def display_release_notes_link(latest_version: str) -> None:
+    """Provide a link to the latest CumulusCI Release Notes"""
+    release_notes_link = (
+        f"https://github.com/SFDO-Tooling/CumulusCI/releases/tag/v{latest_version}"
+    )
+    console = rich.get_console()
+    console.print(
+        f"See the latest CumulusCI Release Notes: [link={release_notes_link}]{release_notes_link}[/link]"
+    )
+
+
 @click.group("main", help="")
+@click.version_option(show_version_info(), message="")
```

Comment:
This doesn't look right. It runs show_version_info() at import time, so it is displayed for every invocation of cci whether or not `--version` was specified.

Reply:
good catch 👍